src/coreclr/jit/emit.h

```diff
@@ -885,8 +885,8 @@ class emitter
 
         ////////////////////////////////////////////////////////////////////////
         // Small constant size (with/without prev offset, assuming host==target):
-        // x86:         11/15 bits
-        // amd64:       10/15 bits
+        // x86:         8/12 bits
+        // amd64:       7/12 bits
```

Comment:
Shouldn't this be the following, since we're only using 1 extra bit now:
```suggestion
        // x86:         10/14 bits
        // amd64:       9/14 bits
```